starter-code/src/app/my-home/my-home.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { MoviesService } from '../movies.service';
+import { ActivatedRoute } from '@angular/router';
+import { Movie } from '../movies-data';
+
+@Component({
+  selector: 'app-my-home',
+  templateUrl: './my-home.component.html',
+  styleUrls: ['./my-home.component.css']
+})
+export class MyHomeComponent implements OnInit {
+  movieList: Movie[];
+  movieId: number;
```

Comment:
Careful with making IDs as numbers. Usually they would be strings or their own specific object :)